views/partials/navigation.hbs

```diff
@@ -1,6 +1,10 @@
 <nav class="navigation">
     <button class="navigation__backButton__container navigation__link go-back">
-        <img class="navigation__icon" src="/public/assets/arrow-left.svg" alt="Go back to {{previousPageName}}" data-previous-page-path="{{ previousPagePath }}" />
+        <img class="navigation__icon" src="/public/assets/arrow-left.svg"
+             alt="Go back to {{previousPageName}}"
+             data-previous-page-path="{{ previousPagePath }}"
+             data-close-on-back="{{ closeOnBack }}"
```

Comment:
//super nit.... but since were here, remove the extra spaces `{{ closeOnBack }}{{ closeOnBack }}`

Reply:
Removed the space